app/models/patient.rb

```diff
@@ -55,12 +66,16 @@ class Patient
   field :household_size_adults, type: Integer
   field :insurance, type: String
   field :income, type: String
+  field :special_circumstances, type: Array, default: []
```

Comment:
Looking at the relocated field names, this is the only one that strikes me as becoming unintuitive when moved from belonging to a "pregnancy" to belonging to a "patient". Are we talking about special circumstances wrt the pregnancy, the patient, or either? If either, then I think we've actually improved the semantic meaning here. :100: